core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -64,9 +84,45 @@ oppia.factory('CollectionRightsBackendApiService', [
          */
         setCollectionPrivate: function(collectionId, collectionVersion) {
           return $q(function(resolve, reject) {
-            _setCollectionStatus(
+            SetCollectionStatus(
               collectionId, collectionVersion, false, resolve, reject);
           });
+        },
+
+        /**
+         * Adds a a new owner to the collection, given its ID and version.
+         */
+        SetCollectionOwner: function(
+          collectionId, collectionVersion, newCollectionOwner) {
```

Comment:
newCollectionOwner --> newOwnerUsername

(better to provide clarity on whether it's username/id; "collection" is already implicit from context)

Ditto below.